sql/atxs/atxs_test.go

```diff
@@ -1727,10 +1726,10 @@ func Test_Previous(t *testing.T) {
 		for range 10 {
 			previousAtxs = append(previousAtxs, types.RandomATXID())
 		}
-		// used by 50 IDs randomly
-		for range 50 {
-			prev := previousAtxs[rand.Intn(len(previousAtxs))]
-			index := slices.Index(previousAtxs, prev)
+		// used by 100 IDs randomly
+		for range 100 {
+			index := rand.IntN(len(previousAtxs))
+			prev := previousAtxs[index]
```

Comment:
How about making the test predictable instead? So there is 0% chance of it failing because of bad luck.

Reply:
removed randomness